105-server-side-apply-implementation-fg-timelines.md

```diff
@@ -0,0 +1,65 @@
+# Server Side Apply - Implementation Split, Feature Gates, and Timelines
+
+This proposal builds on the original [Server Side Apply proposal](052-k8s-server-side-apply.md), which outlines the technical specification and behavior of Kubernetes Server Side Apply (SSA).
+
+Server Side Apply enables declarative updates to Kubernetes resources by tracking ownership of individual fields through the `.metadata.managedFields` property. 
+This allows multiple controllers (such as Strimzi and other operators like Kyverno) to safely modify different parts of the same resource without overwriting each other’s changes.
+
+With Server Side Apply:
+- Strimzi will explicitly claim ownership only of the fields it manages.
+- Other tools can safely annotate or patch resources without triggering reconciliation loops.
+- Reconciliation is simplified because Strimzi no longer needs to diff the entire resource or filter changes manually.
+
+Strimzi will adopt the **reconstructive controller pattern** recommended by Kubernetes, meaning each resource will be fully rebuilt from desired state and applied via Server Side Apply in every reconciliation loop. 
```

Comment:
Any link to what `reconstructive controller pattern` is and where it is recommended? Google did not really found anything.

Reply:
Yeah I guess I just copied it from somewhere. I changed the sentence and removed this.